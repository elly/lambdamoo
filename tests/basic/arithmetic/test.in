1 + 1
4 - 2
5 * 7
8 - 11
12 / 6
12 / 7
11.0 - 5.5
(2 + 2) * 3
sqrt(4.0)
