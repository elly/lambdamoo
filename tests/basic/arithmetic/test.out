2
2
35
-3
2
1
5.5
12
2.0
